commands/membercount.js

```diff
@@ -0,0 +1,50 @@
+const { SlashCommandBuilder } = require('@discordjs/builders')
+const { reply } = require('../helpers/interactionHelper')
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('membercount')
+    .setDescription('Telt het aantal leden in deze discord.')
+    .addBooleanOption((useroption) =>
+      useroption.setName('online').setDescription('toon enkel online gebruikers [optioneel]')
+    )
+    .addChannelOption((useroption) =>
+      useroption.setName('channel').setDescription('van welk kanaal wil je het aantal zien [optioneel]')
+    ),
+  async execute (interaction) {
+    const channel = interaction.options.getChannel('channel')
+    const onlyShowOnline = interaction.options.getBoolean('online')
+    // get either the members of a channel, or from the whole server
+    const members = (await channel?.fetch())?.members ?? await interaction.guild.members.fetch()
+
+    const memberCount = members.filter((member) => {
+      if (member.user.bot) {
+        // don't count bots in any case
+        return false
+      }
+
+      if (onlyShowOnline === null) {
+        // if onlyShowOnline is not provided => always count
+        return true
+      } else {
+      // if onlyShowOnline is provided, then count only where the status matches the requested online status
+        if (onlyShowOnline) {
+          return member.presence?.status && member.presence.status !== 'offline'
+        } else {
+          return !member.presence?.status || member.presence.status === 'offline'
+        }
+      }
+    }).size
+    const channelOrServer = channel || interaction.guild.name
+
+    // crude pluralization below, can eventually be fixed when/if decent i18n support is added
```

Comment:
Could be nice to add a 'todo' in this comment, as some IDE's might pick it up in some overviews?
Minor comment though, won't block the PR based on this alone!